### PR TITLE
Fix parsing of issue resolution

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: src/SonarQube.Client/Services/SonarQubeService.cs [hunk 1]
```diff
@@ -236,12 +236,7 @@ public Uri GetProjectDashboardUrl(string projectKey)
             allIssuesResult.EnsureSuccess();
 
             return allIssuesResult.Value
-                .Where(x =>
-                {
-                    var resolutionState = SonarQubeIssue.ParseResolutionState(x.Resolution);
-                    return resolutionState == SonarQubeIssueResolutionState.WontFix ||
-                        resolutionState == SonarQubeIssueResolutionState.FalsePositive;
-                })
+                .Where(x => SonarQubeIssue.ParseResolutionState(x.Resolution) != SonarQubeIssueResolutionState.Unresolved)
```
Comment (Evangelink) [inline src/SonarQube.Client/Services/SonarQubeService.cs]: We could also do `x => !string.IsEmpty(x.Resolution)`

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SonarQubeService.cs#L87](https://github.com/SonarSource/sonarlint-visualstudio/blob/dc1806877cf9e7c3f6b4fb30824c3c5a8792ddfb/src/SonarQube.Client/Services/SonarQubeService.cs#L87): 'System.Exception' should not be thrown by user code. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS112)
1. ![INFO][INFO] [SonarQubeService.cs#L87](https://github.com/SonarSource/sonarlint-visualstudio/blob/dc1806877cf9e7c3f6b4fb30824c3c5a8792ddfb/src/SonarQube.Client/Services/SonarQubeService.cs#L87): Complete the task associated to this 'TODO' comment. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SonarQubeService.cs#L87](https://github.com/SonarSource/sonarlint-visualstudio/blob/dc1806877cf9e7c3f6b4fb30824c3c5a8792ddfb/src/SonarQube.Client/Services/SonarQubeService.cs#L87): 'System.Exception' should not be thrown by user code. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS112)
1. ![INFO][INFO] [SonarQubeService.cs#L87](https://github.com/SonarSource/sonarlint-visualstudio/blob/dc1806877cf9e7c3f6b4fb30824c3c5a8792ddfb/src/SonarQube.Client/Services/SonarQubeService.cs#L87): Complete the task associated to this 'TODO' comment. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/SonarQube.Client/Services/SonarQubeService.cs (hunk 1 above)
Comment (duncanp-sonar) [reply src/SonarQube.Client/Services/SonarQubeService.cs]: I'd leave it like this: it's clearer what the expected values are.

**Pull request merged**